Description: Remove -Wstrict-prototypes build flag from libnyquist.
 The build flag -Wstrict-prototypes produces many warnings and we do not want to
 modify the upstream nyquist code to just remove compiler warnings.
Author: Benjamin Drung <bdrung@debian.org>
Origin: upstream, r13159

--- a/lib-src/libnyquist/configure.ac
+++ b/lib-src/libnyquist/configure.ac
@@ -86,9 +86,6 @@
 dnl should add to the compiler we are using.
 dnl --------------------------------------------------------------------------
 
-dnl Strict prototypes flags for C (only C because doesn't work for C++)
-AX_CFLAGS_STRICT_PROTOTYPES(CFLAGS)
-
 dnl Sensible warnings for C
 AX_CFLAGS_WARN_ALL(wall_flags)
 CFLAGS="${CFLAGS} $wall_flags"
